packager/mpd/public/mpd_params.h

```diff
@@ -83,6 +83,10 @@ struct MpdParams {
   /// If enabled, allow switching between different codecs, if they have the
   /// same language, media type (audio, video etc) and container type.
   bool allow_codec_switching = false;
+  /// If enabled, PlayReady Object <mspr:pro> will be inserted into
+  /// <ContentProtection ...> element alongside with <cenc:pssh>
+  /// when using PlayReady protection system
+  bool allow_mspr_pro = false;
 };
 
```

Comment:
Can you add a unit-test in mpd_utils_unittest.cc like ContentProtectionMarlin?

Reply:
Sure

Reply:
Added two unit tests:
ContentProtectionPlayReadyCencMspr
ContentProtectionPlayReadyCenc